pkg/services/libraryelements/libraryelements_test.go

```diff
@@ -333,3 +337,8 @@ func getCompareOptions() []cmp.Option {
 		}),
 	}
 }
+
+func mockRequestBody(v interface{}) io.ReadCloser {
+	b, _ := json.Marshal(v)
+	return io.NopCloser(bytes.NewReader(b))
+}
```

Comment:
You probably seen this before, but no big harm to duplicate some test code.
If the API package is to be refactored into smaller ones - this would become a common internal testing utility function, I guess.